src/khoj/interface/web/content_type_input.html

```diff
@@ -113,6 +113,8 @@ <h2 class="section-title">
             suffix = "**/*.org"
         else if ('{{content_type}}' === "pdf")
             suffix = "**/*.pdf"
+        else if ('{{content_type}}' === "plaintext")
+            suffix = "**/*.txt"
```

Comment:
Shouldn't we set the suffix be set to all the supported plain text files specified in `plaintext_to_jsonl`?

Reply:
Changed to just use `*` for the file path and let the `plaintext_to_jsonl` file handle filtering.